data/json/npcs/refugee_center/surface_refugees/NPC_Alonso_Lautrec.json

```diff
@@ -200,10 +235,28 @@
       "type": "general",
       "context": "none",
       "value": "asked",
-      "no": "Alonso does not wish to talk about the past, only the future.  There are dark days ahead, but perhaps together we can bring a little light?",
+      "no": "Alonso has many a tale to tell, all slow and drawn-out like, a sensual seduction of the -- how you say?  Ears.  There are dark days ahead, but perhaps together we can bring a little light?  Or at least a little heat.  This is a science joke I make for you.  Friction, it is the joke.",
       "yes": "I'm tryin' ta forget, y'know?  Don't like thinkin' about the past.  Better to plan for the future, even if the future looks grim."
     },
     "responses": [
+      { "text": "Why aren't you wearing any pants?", "topic": "TALK_REFUGEE_Alonso_Pants1" },
+      { "text": "What were you saying before?", "topic": "TALK_NONE" },
+      { "text": "I'd better get going.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_REFUGEE_Alonso_Pants1",
+    "dynamic_line": {
+      "npc_has_var": "Alonso_accent",
+      "type": "general",
+      "context": "none",
+      "value": "asked",
+      "no": "Why were you looking though?  I kid, I know she is hard to miss!  In English, the penis she is a woman no?  No.  I see.  And so do you it seems.  Yes.  There was such a clown, a clown of enormous physical beauty.  The pale, perfect skin, the ruby red lips, amazing hair, and what a sense of humor.  I laugh still, though I have lost everything.  My pants, my money, my many illegal imported triple magnum condoms I stole from a priest.  All lost to that one beautiful night.  The candles, the balloons, the singing.  It was the most romantic childrens' birthday party you could hope to imagine.  I would do it all again, and there was so much we did.  It was the end of that party, but the beginning of something beautiful, for several hours in that open minivan we found.  But you do not want to hear of such pleasures, I do not want to get you excited, for what can I do when I look so foolish, and lack the protection needed for more serious excitement.  Though and you must understand, the non-penetrative sex is good and fine too, and Alonso's tongue is good for more than just the telling of long and saucy stories yes?",
```

Comment:
I would suggest that Alonso should still be wearing underclothes. He might refer instead to his "package", which IMO is funnier anyway. It's hard to imagine the other refugees tolerating him standing around fully disrobed in the men's bathroom.